include/megaapi_impl.h

```diff
@@ -2406,7 +2406,7 @@ class MegaApiImpl : public MegaApp
         //Sync
         int syncPathState(string *path);
         MegaNode *getSyncedNode(const LocalPath& path);
-        void syncFolder(const char *localFolder, const char *name, MegaHandle megaHandle, MegaRegExp *regExp = NULL, MegaRequestListener* listener = NULL);
+        void syncFolder(const char *localFolder, const char *name, MegaHandle megaHandle, SyncConfig::Type backupType, MegaRegExp *regExp = NULL, MegaRequestListener* listener = NULL);
```

Comment:
It might be convenient to default the new parameter to a value, i.e. `SyncConfig::TYPE_TWOWAY`

Reply:
I've intentionally make it mandatory in order to avoid setting `TWOWAY` by mistake for a backup